tutorial/build-run.md

```diff
@@ -34,44 +34,56 @@ To initialize configuration and a `genesis.json` file for your application and a
 
 > _*NOTE*_: If you have run the tutorial before, you can start from scratch with a `nsd unsafe-reset-all` or by deleting both of the home folders `rm -rf ~/.ns*`
 
-> _*NOTE*_: If you have the Cosmos app for ledger and you want to use it, when you create the key with `nscli keys add jack` just add `--ledger` at the end. That's all you need. When you sign, `jack` will be recognized as a Ledger key and will require a device. 
+> _*NOTE*_: If you have the Cosmos app for ledger and you want to use it, when you create the key with `nscli keys add jack` just add `--ledger` at the end. That's all you need. When you sign, `jack` will be recognized as a Ledger key and will require a device.
 
 ```bash
 # Initialize configuration files and genesis file
-nsd init --chain-id namechain
+  # moniker is the name of your node
+nsd init <moniker> --chain-id namechain
 
-# Copy the `Address` output here and save it for later use 
-# [optional] add "--ledger" at the end to use a Ledger Nano S 
+
+# Copy the `Address` output here and save it for later use
+# [optional] add "--ledger" at the end to use a Ledger Nano S
 nscli keys add jack
 
 # Copy the `Address` output here and save it for later use
 nscli keys add alice
 
 # Add both accounts, with coins to the genesis file
-nsd add-genesis-account $(nscli keys show jack -a) 1000nametoken,1000jackcoin
-nsd add-genesis-account $(nscli keys show alice -a) 1000nametoken,1000alicecoin
+nsd add-genesis-account $(nscli keys show jack -a) 1000nametoken,100000000stake
+nsd add-genesis-account $(nscli keys show alice -a) 1000nametoken,100000000stake
 
 # Configure your CLI to eliminate need for chain-id flag
 nscli config chain-id namechain
 nscli config output json
 nscli config indent true
 nscli config trust-node true
+
+nsd gentx --name <key_name>
 ```
 
+After you have generated you a genesis transcation, you will have to input the gentx into the genesis file, so that your nameservice chain is aware of the validators. To do so, run:
+
+`nsd collect-gentxs`
+
+and to make sure your genesis file is correct, run:
+
+`nsd validate-genesis`
+
 You can now start `nsd` by calling `nsd start`. You will see logs begin streaming that represent blocks being produced, this will take a couple of seconds.
```

Comment:
Following these exact steps yields a panic: https://pastebin.com/ZT0Z2YVU

Reply:
IT WORKS HEHE